airflow/www/extensions/init_appbuilder.py

```diff
@@ -212,6 +212,9 @@ def init_app(self, app, session):
         auth_manager = create_auth_manager()
         auth_manager.appbuilder = self
         auth_manager.init()
+        # FAB auth manager is used in both application AF2 and AF3. It can behave differently depending
+        # on the application. Setting this flag so that it knows this instance is run in AF2
+        auth_manager.airflow_2 = True
```

Comment:
```suggestion
        # FAB auth manager is used in both the old FAB UI and the new React UI. It can behave differently depending
        # on the application. Setting this flag so that it knows this instance is run in FAB.
        auth_manager.is_in_fab = True
```

Let's be more clear it's FAB vs React.

Reply:
I would be more accurate and say FAB vs Fastapi. This code is executed on the back-end so let's compare the backends. Plus, there is also some React in the old UI

Reply:
Updated